client/generated_node_input.go

```diff
@@ -1,79 +0,0 @@
-package client
```

Comment:
It seems that you can not delete this

Reply:
^^ You may miss this comment

Reply:
I added it back. But I am not sure why the script doesn't recreate this file. Also, not sure why we need this file

Reply:
We can check it later. Let's retain it now.